xml/obs_moderation.xml

```diff
@@ -51,6 +51,23 @@
     </figure>
   </sect1>
 
+  <sect1>
+    <title>Setup Notifications</title>
+    <para>
+      To make use of the full potential of the moderation feature in OBS, it is recommended to subscribe all users and moderators to notifications produced as soon as content gets reported or moderated.
+      Admins can setup default subscriptions for the events related to moderated or reported content, in order to enable notifications for all users and moderators of an OBS instance.
```

Comment:
```suggestion
      Admins can configure default subscriptions for events related to moderated or reported content, enabling notifications for all users and moderators in an OBS instance.
```
I find this sentence a little bit convoluted... What about this suggestion?

Reply:
sounds good :+1: